### DISCUSSION TITLE
deck.gl and WKT polygons

### DISCUSSION THREAD

**Discussion by fprivitera** [apache/superset#18458] [Ideas]
**Is your feature request related to a problem? Please describe.**
there is a missing opportunity of visualize  a lot of poligons. Most of OpenGIS databases handle WKT polygons, not geojsons or polylines.

for instance: https://trino.io/docs/311/functions/geospatial.html

**Describe the solution you'd like**
add WKT as new type of "lines encoding"

<img width="547" alt="Screenshot 2021-03-10 at 17 11 17" src="https://user-images.githubusercontent.com/635723/110660634-086f0a00-81c4-11eb-962d-1a1f794a69e7.png">

**Comment by mjj203** [comment]
I was just about to submit an issue for this exact issue. WKT point, polygon, and linestring should all be supported for any of the deckGL spatial visualizations.

**Comment by junlincc** [comment]
tracking related issue https://github.com/apache/superset/issues/13573 here

**Comment by jo-chemla** [comment]
Also interested, especially since I cannot use Postgis functions in the SQL Lab/custom dataset fields.
`No function matches the given name and argument types` when using `ST_centroid` for example. 
Same error returned when executing `SELECT PostGIS_Full_Version()`. 

I've installed Superset via the docker-compose flow.
